.github/workflows/ci.yml

```diff
@@ -15,6 +15,8 @@ env:
   # List of packages that can not target Wasm.
   # `vello_tests` uses `nv-flip`, which doesn't support Wasm.
   NO_WASM_PKGS: "--exclude vello_tests --exclude simple_sdl2"
+  # The files stored in LFS the tests need to access
+  LFS_FILES: "vello_tests/snapshots/*.png"
```

Comment:
I'm not sure how well this will handle adding additional files.
I think the approach is to add a comma within the quotes, which I think might work in both places. But it's a bit unclear

Reply:
We can cross that bridge when we actually need to. It's just a question of style - there are multiple possible solutions. 👍

Reply:
@xStrom we need to cross this bridge in #832
I'm a bit lost

Reply:
The issue is isomorphic to https://github.com/orgs/community/discussions/25761#discussioncomment-6508758, I believe

Reply:
Aha, that soon. I can look at it later today and/or tomorrow if you guys haven't solved it before then.